src/SourceBuild/content/repo-projects/aspnetcore.proj

```diff
@@ -26,6 +26,7 @@
     <RepositoryReference Include="arcade" />
     <RepositoryReference Include="runtime" />
     <RepositoryReference Include="xdt" />
+    <RepositoryReference Include="efcore" />
```

Comment:
I think this needs to be commented out initially so that the sources get synced into the VMR and then enabled in the VMR build via a subsequent PR.

Reply:
I've opened https://github.com/dotnet/installer/pull/19432 to do the syncing first.